algorithms/maximum_sum_array.py

```diff
@@ -0,0 +1,12 @@
+class max_sum_array(object):
+   def maxSubArray(self, nums):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396358)